commit cf0ca8d52e1653d4aa4311a4649af8dc541ce6b4
Author: Joseph Myers <josmyers@redhat.com>
Date:   Mon May 20 13:10:31 2024 +0000

    Update syscall lists for Linux 6.9
    
    Linux 6.9 has no new syscalls.  Update the version number in
    syscall-names.list to reflect that it is still current for 6.9.
    
    Tested with build-many-glibcs.py.

    Modified for RHEL by: Patsy Griffin <patsy@redhat.com>

diff --git a/sysdeps/unix/sysv/linux/syscall-names.list b/sysdeps/unix/sysv/linux/syscall-names.list
index 6557bcfde4..672d39eaad 100644
--- a/sysdeps/unix/sysv/linux/syscall-names.list
+++ b/sysdeps/unix/sysv/linux/syscall-names.list
@@ -21,8 +21,8 @@
 # This file can list all potential system calls.  The names are only
 # used if the installed kernel headers also provide them.
 
-# The list of system calls is current as of Linux 6.8.
-kernel 6.8
+# The list of system calls is current as of Linux 6.9.
+kernel 6.9
 
 FAST_atomic_update
 FAST_cmpxchg
